litgpt/utils.py

```diff
@@ -47,9 +47,10 @@ def num_parameters(module: nn.Module, requires_grad: Optional[bool] = None) -> i
     return total
 
 
-def check_valid_checkpoint_dir(checkpoint_dir: Path) -> None:
+def check_valid_checkpoint_dir(checkpoint_dir: Path, lora: bool = False) -> None:
+    model_filename = "lit_model.pth.lora" if lora else "lit_model.pth"
```

Comment:
If we expect to have more of these in the future (like adapter or dora), it would be better to add `model_filename="lit_model.pth"` as the argument instead of `lora=`

Reply:
hm the test ran fine locally, I'll double-check